### ISSUE TITLE
offsets?

### ISSUE THREAD

**Issue by xxA1eksandrxx** [3a1/Evelion#2]
Hi! The great progect. 
Please tell me what the offsets are:
hw + 0xEC9780
hw + 0x120461C - 0x0250
client + 0x100DF4

**Comment by 3a1** [comment]
Hi, thx
#define dwViewMatrix 0xEC9780
#define dwEntityList 0x120461C

#define m_vecOrigin  0x184
#define m_stateValue 0x17C
#define m_entitySize 0x250

client + 0x100DF4 is offset that check if player is in the match or not
if you have any questions just ask

**Comment by xxA1eksandrxx** [comment]
where can I get the offsets for the new version, mb dumper cs1.6? I think it's 9...

**Comment by xxA1eksandrxx** [comment]
hw + 0x105CFC8? ;)

**Comment by 3a1** [comment]
if you mean the new steam version, then i dont know myself, the new steam version is not working for me so i cannot even check

**Comment by xxA1eksandrxx** [comment]
> hw + 0x105CFC8? ;)

and how did you look for these offsets? there is no dumper for the 1.6

**Comment by 3a1** [comment]
yeah i find them myself, for example entity list, i know what he looks like, i know that first byte in entity list is model info, name info etc. also before entity list is always map name. by these factors i can find the entity list and then all other offsets

**Comment by xxA1eksandrxx** [comment]
this offsets is not corrected?:
https://github.com/KleskBY/CS-1.6-External-cheat?ysclid=m08jofq6q6144800744

**Comment by 3a1** [comment]
i dont know, but as i give a try to play on KleskBy cheat, even on the cs 1.6 version that it supposed to work on, it doesnt work, i think for now its just outdated

**Comment by xxA1eksandrxx** [comment]
hw + 0x105CFC8?

**Comment by 3a1** [comment]
probably one more lobby check

**Comment by xxA1eksandrxx** [comment]
is it possible to run in full-screen mode?

**Comment by 3a1** [comment]
for this cheat no, but im now making new one that will be full kernel and will support fullscreen esp, i will release it in next days

**Comment by xxA1eksandrxx** [comment]
we need to look for offsets for the new version somehow...

**Comment by 3a1** [comment]
but its working on steam version , just that new one is not
you need choose steam legacy - pre-25th anniversary build in cs 1.6 properties and it will work

**Comment by xxA1eksandrxx** [comment]
discord?

**Comment by 3a1** [comment]
i dont use discord, only telegram @ZebraEth
